libs/langchain-mongodb/tests/integration_tests/conftest.py

```diff
@@ -10,25 +10,25 @@
 from pymongo import MongoClient
 
 
-@pytest.fixture(scope="session")
+@pytest.fixture
 def technical_report_pages() -> List[Document]:
     """Returns a Document for each of the 100 pages of a GPT-4 Technical Report"""
     loader = PyPDFLoader("https://arxiv.org/pdf/2303.08774.pdf")
     pages = loader.load()
     return pages
 
 
-@pytest.fixture(scope="session")
+@pytest.fixture
 def connection_string() -> str:
     return os.environ["MONGODB_URI"]
 
 
-@pytest.fixture(scope="session")
```

Comment:
What's the rationale here? We want to create the client for each test?

Reply:
Otherwise we can't test the `close()` method.

Reply:
You can't use a client after it is closed